mlflow/server/js/src/common/forms/validations.js

```diff
@@ -27,6 +27,27 @@ export const getExperimentNameValidator = (getExistingExperimentNames) => {
   };
 };
 
+export const getExperimentIdValidator = (getCurrentExperimentId, getExistingExperimentIds) => {
+  return (rule, value, callback) => {
+    if (value.length === 0) {
+      // no need to execute below validations when no value is entered
+      // eslint-disable-next-line callback-return
+      callback(undefined);
+    } else if (isNaN(value)) {
+      callback(`Experiment ID must be an integer`);
```

Comment:
shouldn't this validate against string? perhaps just rely on GetExpById()'s response to validate?

Reply:
that sounds good. would also validate against floats.

Reply:
Upon some more thought. Changed to an integer check. By default the form gives string values, which does work for the API call.

Correct me if I'm wrong, `getExperiment` method would also get deleted experiment. It probably better to not include those for the move target.

Reply:
Ah I see, that makes sense. I think as long as I remove the integer condition it should validate string. It just checks that the Id is not the current experiment id and that it is in the list of existing experiment ids.